package.json

```diff
@@ -50,6 +50,7 @@
     "@tailwindcss/forms": "^0.5.7",
     "@tailwindcss/line-clamp": "^0.4.4",
     "@tailwindcss/typography": "^0.5.10",
+    "@types/react-text-truncate": "^0.14.4",
```

Comment:
Make this as a part of `devDependencies`.

Reply:
We cannot move `@types` deps to `devDependencies`, otherwise the Netlify Build would fail

Reply:
(https://app.netlify.com/sites/asyncapi-website/deploys/65e5e825663a7f0008192807)

Reply:
Check the latest commit.

Reply:
I think then we can move all other `@types*` dependencies to `devDependencies` now